hisi-idt.py

```diff
@@ -243,27 +243,19 @@ def main(argv):
             img1 = arg
         elif opt in ("--img2"):
             img2 = arg
-    if dev == '':
-        if os.path.exists(dev1):
-            dev = dev1
-        elif os.path.exists(dev2):
-            dev = dev2
-        else:
-            print 'Device not detected under /dev/serial/by-id/. Please use -d.'
-            sys.exit(3)
```

Comment:
That was a mistake, I don't know how it happened:

 https://github.com/96boards/burn-boot/pull/5